clang/lib/CIR/Dialect/IR/CIRDialect.cpp

```diff
@@ -2384,12 +2410,16 @@ cir::CallOp::verifySymbolUses(SymbolTableCollection &symbolTable) {
 
 ::mlir::ParseResult CallOp::parse(::mlir::OpAsmParser &parser,
                                   ::mlir::OperationState &result) {
-  return parseCallCommon(parser, result);
+
+  return parseCallCommon(parser, result, getExtraAttrsAttrName(result.name));
 }
 
 void CallOp::print(::mlir::OpAsmPrinter &state) {
   mlir::Value indirectCallee = isIndirect() ? getIndirectCall() : nullptr;
-  printCallCommon(*this, indirectCallee, getCalleeAttr(), state);
+  printCallCommon(*this, indirectCallee, getCalleeAttr(), state,
+    [this]() -> ::mlir::cir::ExtraFuncAttributesAttr {
+      return getExtraAttrs();
```

Comment:
As I mentioned in previous reviews, these lambdas are unnecessary, there are more simple and clean ways to go about this.